src/autocomplete/popup.js

```diff
@@ -153,7 +153,7 @@ class AcePopup {
                 el.setAttribute("aria-activedescendant", ariaId);
                 selected.setAttribute("role", optionAriaRole);
                 selected.setAttribute("aria-roledescription", nls("item"));
-                selected.setAttribute("aria-label", popup.getData(row).value);
+                selected.setAttribute("aria-label", popup.getData(row).caption);
```

Comment:
`caption` is an optional prop to be set ([types](https://github.com/ajaxorg/ace/blob/master/ace.d.ts#L340)), should be fall back to `value` if caption is undefined? (or do we always set `popup.getData(row).caption` somewhere?)